addons/im_livechat/models/digest.py

```diff
@@ -15,36 +15,32 @@ class Digest(models.Model):
     kpi_livechat_response_value = fields.Float(digits=(16, 2), compute='_compute_kpi_livechat_response_value')
 
     def _compute_kpi_livechat_rating_value(self):
-        channels = self.env['mail.channel'].search([('livechat_operator_id', '=', self.env.user.partner_id.id)])
-        for record in self:
-            start, end, company = record._get_kpi_compute_parameters()
-            domain = [
-                ('create_date', '>=', start), ('create_date', '<', end),
-                ('rated_partner_id', '=', self.env.user.partner_id.id)
-            ]
-            ratings = channels.rating_get_grades(domain)
-            record.kpi_livechat_rating_value = ratings['great'] * 100 / sum(ratings.values()) if sum(ratings.values()) else 0
+        channels = self.env['mail.channel'].search([('channel_type', '=', 'livechat')])
```

Comment:
Previous code seemingly relied on the "current partner", was that incorrect?

As we are modifying the logic here and not just "batch computing", should we make a separate commit? Maybe as a backported bugfix?